pyomo/checker/tests/test_examples.py

```diff
@@ -23,7 +23,6 @@
 
 def createTestMethod(defs, package, checkerName, key):
     def testMethod(obj, name):
-        import pyomo.environ
```

Comment:
I'm not sure - but it didn't break anything when removed. Re-add, or no?

Reply:
My guess is to re-add.  The issue is that generally `pyomo.environ` must be imported so that factories (like the `SolverFactory` or `TransformationFactory`) are correctly set up.  When you run the whole test suite, generally you don't have to worry about the import, because some test before you already imported it.  However, if you ever run the test in isolation, you can start seeing test failures (because the other test that used to run before you an import `pyomo.environ` is no longer running).